### ISSUE TITLE
Install fails on OSX because of -D option

### ISSUE THREAD

**Issue by ryanbahneman** [in0rdr/diary#41]
Here is the error I'm seeing:

```zsh
--> make install
cp diary /usr/local/bin/diary
install -D -m 644 diary.1 /usr/local/share/man/man1/diary.1
install: illegal option -- D
usage: install [-bCcpSsv] [-B suffix] [-f flags] [-g group] [-m mode]
               [-o owner] file1 file2
       install [-bCcpSsv] [-B suffix] [-f flags] [-g group] [-m mode]
               [-o owner] file1 ... fileN directory
       install -d [-v] [-g group] [-m mode] [-o owner] directory ...
make: *** [install] Error 64
```

Converting the offending lines in the makefile to the following solves the issue:
```
install -d $(MANDIR)/man1
install -m644 $(MAN1) $(MANDIR)/man1/$(MAN1)
```

**Comment by in0rdr** [comment]
Thanks, should be resolved in last commit: [c30b70b](https://github.com/in0rdr/diary/commit/c30b70bbd5bc014bf26ce5a0246355e9e52565e3)

**Comment by ryanbahneman** [comment]
The fix worked great.
Thanks!
